tests/unit/070_job_processes.spec.ts

```diff
@@ -0,0 +1,188 @@
+import * as chai from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import { prepareCommands } from '../../src/api/process';
+import { CommandType, Command, CommanyTypePriority } from '../../src/api/config';
```

Comment:
`CommanyTypePriority` I believe you have a typo here.

Reply:
done